### DISCUSSION TITLE
Can the repo be used on a URI ?

### DISCUSSION THREAD

**Discussion by jehoshua7** [FallingLights/Teachable-dl#42] [Q&A]
I have a video when played in Google Chrome, displays the transcript/captions. Have tried various methods to save/download the captions, even tried a few Chrome extensions.

Can this repository be modified so that it doesn't require a login to the Teachable site ?  Just parse a URI, the script opens the video in Chrome and outputs the captions to a (.vtt) file ?

**Comment by FallingLights** [comment]
depends on the site and video player you could always just manually download the vtt file from the chrome network tab

**Reply by jehoshua7** [reply]
@FallingLights  -  I played a local MP4 video in Chrome. As it was playing, the captions appear but there seems no (easy) way to save the captions. In the Chrome network tab, it shows the .mp4 file (3 times) and 7 files of this nature - data:image/svg+xml;base64 , however no VTT files. It could be some filter set or similar, so I will investigate that side of things. Thanks for your help.  :)
